website/static/js/keen.js

```diff
@@ -0,0 +1,143 @@
+var keen = require('keen-js');
```

Comment:
It'll work, but it's still good practice to be explicit about your dependencies.